pkg/api/mlflow/dao/models/app.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"database/sql"
```

Comment:
please organise imports in next way:
```
std packages
empty line
third part packages
empty line
internal pckages
```

Reply:
this will be enforced in https://github.com/G-Research/fasttrackml/pull/155 😄

Reply:
removing this file